Trunk/c_dungeon.py

```diff
@@ -21,106 +21,87 @@ def neighbours(coord, diagonal=False):
 		for xoff, yoff in offsets:
 				yield cx + xoff, cy + yoff
 
-	def empty_neighbours(self, cells, diagonal=False):
+	def empty_neighbours(self, coords, diagonal=False):
```

Comment:
boolean parameters should be obviously named, maybe rename `diagonal` to `check_diagonals`